content/_help/trouble-signing-in/issues-with-authentication-methods._en.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: help
+title: Issues with authentication methods
+category: trouble-signing-in
+permalink: /help/trouble-signing-in/issues-with-authentication-methods/
+order: 3
+---
+
+Depending on the authentication methods you’ve set up, you may still be able to access your Login.gov account. After you’re able to sign in, make sure you’ve set up more than one authentication method to avoid losing access to your account. 
+
+<div class="usa-alert usa-alert--warning margin-bottom-4" role="status">
+  <div class="usa-alert__body">
+    <p class="usa-alert__text">If you cannot sign in with your only authentication method, you will have to delete your account and create a new account. Login.gov cannot unlock your account for you or sign in on your behalf.</p>
+  </div>
+</div>
+
+## Face or touch unlock isn't working
+
+Unless your device and browser supports cloud sync, face or touch unlock only works on the same device and browser where you set it up. You can only use face or touch unlock on multiple devices if they support cloud sync.
```

Comment:
I personally like to start instructions with the positive case and not negating with "unless", just an idae:
```suggestion
If your device and browser supports cloud sync, you can use face or touch unlock on multiple devices. Otherwise, face or touch unlock only works on the same device and browser where you set it up.
```